src/fsharp/FSComp.txt

```diff
@@ -58,6 +58,8 @@ buildCouldNotReadVersionInfoFromMscorlib,"Could not read version from mscorlib.d
 242,buildArgInvalidFloat,"'%s' is not a valid floating point argument"
 243,buildUnrecognizedOption,"Unrecognized option: '%s'"
 244,buildInvalidModuleOrNamespaceName,"Invalid module or namespace name"
+246,invalidResponseFile,"Invalid response file '%s' ( '%s' )"
+258,responseFileNotFound,"Response file '%s' not found in '%s'"
```

Comment:
Probably, though a while back in F# 2.0 we decided to always use a completely new error number and add new messages at the end of the file